app/controllers/api/v1/targets_controller.rb

```diff
@@ -0,0 +1,22 @@
+module Api
+  module V1
+    class TargetsController < Api::V1::ApiController
+      def create
+        authorize Target
+        @target = current_user.targets.create!(target_params)
+
+        if @target.save
+          render json: @target, status: :created, location: [:api, :v1, @target]
```

Comment:
I believe this is not needed, since rails knows where to find it due to the convention 😉 

```suggestion
          render json: @target, status: :created
```